ecl/hql/hqlgram.y

```diff
@@ -6728,6 +6730,30 @@ primexpr1
                             Owned<ITypeInfo> ltype = lexpr->getType();
                             Owned<IHqlExpression> locale = (ltype->getTypeCode() == type_varstring) ? lexpr.getLink() : createValue(no_implicitcast, makeVarStringType(ltype->getStringLen()), lexpr.getLink());
                             $$.setExpr(createValue(no_unicodeorder, makeIntType(4, true), $3.getExpr(), $5.getExpr(), locale.getLink(), $9.getExpr()));
+                         }
+    | LIKELY '(' booleanExpr ')'
+                        {
+                            parser->normalizeExpression($3);
+                            $$.inherit($3);
+                            $$.setPosition($1);
+                        }
+    | LIKELY '(' booleanExpr ',' expression ')'
+                        {
+                            parser->normalizeExpression($3);
```

Comment:
not needed - see above.
Should have
parser->normalizeExpression($5, type_real, true);
and
OwnedHqlExpr probability = $5.getExpr();

otherwise it will currently leak.